rmf_fleet_adapter/CMakeLists.txt

```diff
@@ -262,6 +262,7 @@ target_link_libraries(experimental_lift_watchdog
     ${rclcpp_LIBRARIES}
     ${rmf_fleet_msgs_LIBRARIES}
     ${std_msgs_LIBRARIES}
+    Threads::Threads
```

Comment:
Looking at https://github.com/open-rmf/rmf_traffic/pull/74/files, can you also include a `find_package(Threads)`?